src/app/hooks/useSignatureRequest.ts

```diff
@@ -82,18 +85,17 @@ export function useSignMessage(messageType: SignatureMessageType) {
 }
 
 export function useSignBip322Message(message: string, address: string) {
-  const { accountsList, seedPhrase, network } = useWalletSelector();
-  return useCallback(
-    async () =>
-      signBip322Message({
-        accounts: accountsList,
-        message,
-        signatureAddress: address,
-        seedPhrase,
-        network: network.type,
-      }),
-    [],
-  );
+  const { accountsList, network } = useWalletSelector();
+  const {getSeed} = useSecretKey();
+  return useCallback(async () => {
+    const seedPhrase = await getSeed();
+   return signBip322Message({
+    accounts: accountsList,
+    message,
+    signatureAddress: address,
+    seedPhrase,
+    network: network.type,
+  })}, []);
```

Comment:
This should probably have dependencies listed.

Reply:
no need since no dependencies change for the same request